modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view_{context}"]
+= Customizing a Developer catalog or its sub-catalogs using the YAML view
+
+You can customize a developer catalog by editing the YAML content in the YAML view.
+
+.Prerequisites
+
+* An OpenShift web console session by a cluster administrator.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Administration* -> *Cluster Settings*.
+. Select the *Configuration* tab, click the *Console (operator.openshift.io)* resource and see the *Details* page.
+. Click the *YAML* tab to open the editor and edit the YAML content as needed. 
+.. For example, 
```

Comment:
Required: The formatting here looks wrong in the preview.

I don't think there is a need for the nested list here, so maybe use a contibuation at this point:

```
. Click the *YAML* tab to open the editor and edit the YAML content as needed. 
+ 
For example, to disable a developer catalog type, insert the following snippet, which defines a list of disabled developer catalog resources:
+
[source,yaml]
....
```